### PR TITLE
circular-buffer: Update the tests following the canonical data

### PR DESCRIPTION
**Pull request by elyssonmr**
Update the tests following the `canonical data` from issue #991

### CODE REVIEW

File: exercises/circular-buffer/circular_buffer_test.py [hunk 1]
```diff
@@ -7,40 +7,37 @@
 )
 
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1
```
Comment (N-Parsons) [inline exercises/circular-buffer/circular_buffer_test.py]: Minor formatting point, but can you add a blank line after the version string for consistency?

File: exercises/circular-buffer/circular_buffer_test.py [hunk 2]
```diff
@@ -7,40 +7,37 @@
...
 class CircularBufferTest(unittest.TestCase):
     def test_read_empty_buffer(self):
         buf = CircularBuffer(1)
         with self.assertRaises(BufferEmptyException):
             buf.read()
 
+    def test_read_just_written_item(self):
+        buf = CircularBuffer(1)
+        buf.write('1')
+        self.assertEqual('1', buf.read())
```
Comment (N-Parsons) [inline exercises/circular-buffer/circular_buffer_test.py]: By convention, we use `assertEqual(actual, expected)` for our tests - can you switch the argument order in all of these tests?

You can check the rest of our conventions in the main [README.md](https://github.com/exercism/python/blob/master/README.md).

File: exercises/circular-buffer/circular_buffer_test.py [hunk 3]
```diff
@@ -7,40 +7,37 @@
...
         buf.write('1')
-        buf.write('2')
-        self.assertEqual(buf.read(), '1')
-        buf.write('3')
-        self.assertEqual(buf.read(), '2')
+        with self.assertRaises(BufferFullException):
+            buf.write('2')
 
     def test_alternate_write_and_read(self):
         buf = CircularBuffer(2)
```
Comment (N-Parsons) [inline exercises/circular-buffer/circular_buffer_test.py]: Looks like this test has been changed in the canonical data - it should be a `CircularBuffer(1)`.

File: exercises/circular-buffer/circular_buffer_test.py (hunk 1 above)
Comment (elyssonmr) [reply exercises/circular-buffer/circular_buffer_test.py]: Done

File: exercises/circular-buffer/circular_buffer_test.py (hunk 2 above)
Comment (elyssonmr) [reply exercises/circular-buffer/circular_buffer_test.py]: Changed then. Thanks to point it out

File: exercises/circular-buffer/circular_buffer_test.py (hunk 3 above)
Comment (elyssonmr) [reply exercises/circular-buffer/circular_buffer_test.py]: Changed that. Thanks

**Comment by elyssonmr** [comment]
@N-Parsons can you check again? Thanks

**Comment by N-Parsons** [comment]
Looks good. Thanks, @elyssonmr!

**Pull request merged**